lib/matplotlib/collections.py

```diff
@@ -319,10 +319,12 @@ def _prepare_points(self):
                 xs, ys = vertices[:, 0], vertices[:, 1]
                 xs = self.convert_xunits(xs)
                 ys = self.convert_yunits(ys)
-                paths.append(mpath.Path(np.column_stack([xs, ys]), path.codes))
+                paths.append(
+                    mpath.Path(np.ma.column_stack([xs, ys]), path.codes)
+                )
```

Comment:
I'm not sure this is necessary, since each `path` will be a `Path` object, and the masking is removed at instantiation of the `Path` object
https://github.com/matplotlib/matplotlib/blob/f8d932e2bdb8a5b9d8ff282e76c74b297ed5f981/lib/matplotlib/path.py#L129
So I think, when you get to here, `xs` and `ys` cannot be masked?

Reply:
Ah okay, yeah. That makes sense. Thanks!